crates/epaint/src/text/text_layout_types.rs

```diff
@@ -319,6 +319,17 @@ impl TextFormat {
 
 // ----------------------------------------------------------------------------
 
+/// How to wrap and elide text.
+///
+/// This enum is used in high-level APIs where providing a [`TextWrapping`] is too verbose.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub enum TextWrapMode {
+    Extend,
+    Wrap,
+    Truncate,
```

Comment:
These should have docstrings to explain them fully

Reply:
That was very much my intention 🤦🏻